test/unit/rubygem_searchable_test.rb

```diff
@@ -45,6 +45,7 @@ class RubygemSearchableTest < ActiveSupport::TestCase
         mailing_list_uri:  "http://example.com",
         source_code_uri:   "http://example.com",
         bug_tracker_uri:   "http://example.com",
+        funding_uri:       "http://example.com",
```

Comment:
you will need to update models/concerns/rubygems_searchable.rb for this.

ps: ignore my prev comment (now deleted)

Reply:
> you will need to update models/concerns/rubygems_searchable.rb for this.

this is also not correct :sweat_smile: this test is outdated and is using linkset fallback. we need to set metadata in version 1.0.1 in setup.